plugins/inputs/opcua/opcua_test.go

```diff
@@ -23,44 +25,70 @@ type OPCTags struct {
 	Want           interface{}
 }
 
-const servicePort = "4840"
+func MapOPCTag(tags OPCTags) (out input.NodeSettings) {
+	out.FieldName = tags.Name
+	out.Namespace = tags.Namespace
+	out.IdentifierType = tags.IdentifierType
+	out.Identifier = tags.Identifier
+	return out
+}
 
 func TestGetDataBadNodeContainerIntegration(t *testing.T) {
 	if testing.Short() {
 		t.Skip("Skipping integration test in short mode")
 	}
 
-	container := testutil.Container{
-		Image:        "open62541/open62541",
-		ExposedPorts: []string{servicePort},
-		WaitingFor:   wait.ForListeningPort(nat.Port(servicePort)),
+	// Spin-up the container
+	ctx := context.Background()
+	req := testcontainers.GenericContainerRequest{
+		ContainerRequest: testcontainers.ContainerRequest{
+			Image:        "open62541/open62541:1.0",
```

Comment:
Looks like 1.0 is latest as well. In other tests I've tried not to call a specific tag so we know as new versions come out when we break things, but in this case maybe it makes sense? Should we be consistent with other opcua integration tests and make this change there too?

Reply:
Yes, I changed it, thanks!